common/views/components/WorkMedia/WorkMedia.js

```diff
@@ -38,12 +37,10 @@ const WorkMedia = ({
             action: 'scroll-to-info:click',
             label: 'scrolled-to-id:work-info'
           }}
-          clickHandler={() => {
-            ReactGA.event({
-              category: 'component',
-              action: 'scroll-to-info:click',
-              label: 'scrolled-to-id:work-info'
-            });
+          trackingEventV2={{
+            eventCategory: 'Control',
+            eventAction: 'scroll to info',
+            eventLabel: id
```

Comment:
This again feels like the control can track it's own use, whereas this might be `WorkMedia, scroll to info`.